packages/core/src/components/QueryBuilder/Rule/Value/Value.tsx

```diff
@@ -0,0 +1,67 @@
+import { memo, useContext } from "react";
+
+import { QueryBuilderContext } from "../../Context";
+import { BooleanValue } from "./BooleanValue";
+import { NumericValue } from "./NumericValue";
+import TextValue from "./TextValue/TextValue";
+
+// import DateTimeValue from "./DateTimeValue";
+
+export interface ValueProps {
+  id: number;
+  attribute: string;
+  operator: string;
+  value: any;
+}
+
+export const Value = ({
+  id,
+  attribute,
+  operator,
+  value: valueProp,
+}: ValueProps) => {
+  const context = useContext(QueryBuilderContext);
+  const { attributes, initialTouched } = context;
+  const value = attribute ? { ...attributes[attribute] } : { type: null };
+  const { type } = value;
+
+  switch (type) {
+    case "boolean": {
+      return (
+        <BooleanValue
+          id={id}
+          value={!!valueProp}
+          // initialTouched={initialTouched}
```

Comment:
Should we uncomment this?

Reply:
the BooleanValue does not receive that prop, I removed it.